OOInteraction/src/commands/CommandHelper.h

```diff
@@ -71,9 +71,9 @@ class CommandHelper
 			auto manager = new Model::TreeManager();
 			manager->setRoot(child);
 
-			auto vis = new Visualization::RootItem(child);
+			auto vis = target->scene()->currentViewItem();
```

Comment:
Actually the line after this one (setPos ...) doesn't make sense now that we have views. So instead of these three lines, just do
target->scene()->currentViewItem()->insertNode(child)